pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -217,10 +210,26 @@ func (c *Operation) QueryConnections(rw http.ResponseWriter, req *http.Request)
 		},
 	}
 
-	err = json.NewEncoder(rw).Encode(response)
+	c.writeResponse(rw, response)
+}
+
+// getQueryParams unmarshals query strings values to interface value
```

Comment:
It would be helpful to describe the v argument. 
You are expecting an annotated struct.

This function appears to populate an annotated struct from the query params.

Reply:
This is simple util function to convert `map[string][]string` to given type v. It doesn't have to be annoted, simple json Unmarshal rule applies here.